Docker/README-multiple-nodes.md

```diff
@@ -0,0 +1,22 @@
+
+# How to run more than the default 4 nodes with docker compose.
+
+The simple steps are:
+
+```
+./gen.sh 6
+
+# That will create a docker-compose.yaml file in ./validators-6/docker-compose.yaml
+
+cd validators-6
+
+docker compose up -d
+docker compose logs -f
+
+```
+
+
+The grafana instance is exposed at http://localhost:3000/
+
+Default user/pass is admin/admin.  You can 'skip' changing that since it's always
```

Comment:
Let's also add a section that explains the `parameters.json`  & `committee.json` file will be auto-generated and if someone would like to edit the `parameters.json` properties could do it via the `gen.sh` file before its creation, or afterwards directly under the `validator-x` folder

Reply:
Only the committee.json file is generated.  The parameters.json is just a copy from what was there before.
If there are parameters that could change in that file, let me know.